app/assets/stylesheets/ama_layout/layout_components/banners.scss

```diff
@@ -18,11 +18,22 @@
     padding: $base-padding;
   }
 
+  &--brand-blue-light{
+    @extend .banner;
+    background: $brand-blue-light;
+    padding: $base-padding;
+  }
+
   &__title,
   &__text{
     color: $white;
   }
 
+  &__text--center{
```

Comment:
Since we talked about helper classes, do we just want to get rid of this one and put the helper class into the html?